src/backend/oracle-data-api/src/test/java/ca/bc/gov/open/jag/tco/oracledataapi/BaseTestSuite.java

```diff
@@ -16,10 +17,14 @@ public class BaseTestSuite {
 
 	@Autowired
 	protected DisputeRepository disputeRepository;
+	
+	@Autowired
+	protected JJDisputeRepository jjDisputeRepository;
 
     @BeforeEach
     protected void beforeEach() throws Exception {
     	disputeRepository.deleteAll();
+    	jjDisputeRepository.deleteAll();

```

Comment:
Probably not in this PR, but we should change this to use @Transactional instead.  This way each unit test rolls back changes.  It also allows us to prepopulate the database with specific test records.  With this deleteAll we can't do that.

Reply:
Yes, good call. Noted for another PR when we have time.